appveyor.yml

```diff
@@ -1,26 +1,36 @@
+build: off
+cache:
+  - '%APPDATA%\npm-cache'
+clone_depth: 1
```

Comment:
Isn't it better to use `shallow_clone`?

Reply:
I did this to enable symlinks: https://github.com/avajs/ava/commit/0d5395731931882527eeaba802dff7f2751dea5a#diff-180360612c6b8c4ed830919bbb4dd459

That's not an issue for this repository, but it's easier to manage these files if they're similar across all repositories in this organization.